sdk/maps/maps-route-rest/README.md

```diff
@@ -96,8 +96,8 @@ const routeDirectionsResult = await client.path("/route/directions/{format}", "j
 });
 
 // You can use the helper function `toColonDelimitedLatLonString` to compose the query string.
-const { toColonDelimitedLatLonString } = require("@azure-rest/maps-route");
-const routeDirectionsResult = await client.path("/route/directions/{format}", "json").get({
```

Comment:
Here I tried to demonstrate that users can either use the helper function or directly compose the query string themselves. So they are two pieces of code. @joheredi do you think it's necessary to change it to let here?

Reply:
@joheredi Any ideas?

Reply:
```diff
+ const { toColonDelimitedLatLonString, isUnexpected } = require("@azure-rest/maps-route");
- const routeDirectionsResult = await client.path("/route/directions/{format}", "json").get({
+ const routeDirectionsResult1 = await client.path("/route/directions/{format}", "json").get({
  queryParameters: {
    query: "51.368752,-0.118332:41.385426,-0.128929",
  },
});
// ...
// You can use the helper function `toColonDelimitedLatLonString` to compose the query string.
- const { toColonDelimitedLatLonString } = require("@azure-rest/maps-route");
- const routeDirectionsResult = await client.path("/route/directions/{format}", "json").get({
+ const routeDirectionsResult2 = await client.path("/route/directions/{format}", "json").get({
  queryParameters: {
    query: toColonDelimitedLatLonString([
      // Origin:
      [51.368752, -0.118332],
      // Waypoints (Optional):
      [45.49735, 9.182435],
      [48.886128, 2.329742],
      [48.159642, 11.518011],
      // Destination:
      [41.385426, -0.128929],
    ]),
  },
});
// ...
- if (isUnexpected(routeDirectionsResult)) {
-  throw routeDirectionsResult.body.error;
+ if (isUnexpected(routeDirectionsResult2)) {
+  throw routeDirectionsResult2.body.error;
}

- routeDirectionsResult.body.routes.forEach(({ summary, legs }) => {
+ routeDirectionsResult2.body.routes.forEach(({ summary, legs }) => {
```

Reply:
@andykao1213 I have updated the code as you said.